docs/development/select-screen.md

```diff
@@ -59,13 +59,97 @@ const screenConfig = {
     }
 }
 ```
+### Choice titles
+
+Configured choices may have text labels drawn with them. These are populated by config, using the "title" and "subtitle" keys. Example:
+
+```json
+{
+    ...
+    "choices": [
+        {
+            "main": "dangermouse",
+            "name": "dangermouseName",
+            "title": "Danger Mouse",
+            "subtitle": "The world's greatest secret agent",
+        },
+    ]
+}          
+```
+
+#### Choice text styling
+
+Choice text elements must have default styling associated with their text elements. Default styles are set within the select screen config, in the object `choicesStyling`, example:
+
+```json
+{
+    ...
+    "choices": [...],
+    "choicesStyling": {
+        "default": {
+            "title": {
+                "style": {
+                    "fontFamily": "ReithSans",
+                    "fontSize": "19px",
+                    "color": "#424242",
+                    "backgroundColor": "#fff",
+                    "fixedWidth": 75,
+                    "align": "center",
+                    "padding": {
+                        "left": 6,
+                        "right": 6,
+                        "top": 2,
+                        "bottom": 2,
+                    },
+                },
+                "position": {
+                    "x": 0,
+                    "y": 53,
+                },
+            },
+            "subtitle": {
+                "style": {
+                    ...
+                },
+                "position": {
+                    ...
+                }
+            }
+        }
+    }
+}
+```
+
+*Both `position` and `style` objects are required for the text to be be displayed.*
+
+Styles for different button states can override the default styles for each of the elements:
+
+```json
+{
+    ...
+    "default": { 
+        ...
+    },
+    "locked": {
+        "title": {
+            "style": {
+                "color": "#fff",
+                "backgroundColor": "#7d4b4b",
+            },
+        },
+        "subtitle": {
+            ...
+        }
+    }
+}
+```
 
 ## How does it find the location of my sprite assets?
 
 - Locate the file `asset-pack-master.json` located inside `themes/your-theme/` (where `your-theme` is the name of your theme).
 - Add your sprite locations and preferred key names to this file.
 
-```javascript
+```json
```

Comment:
Should these code blocks remain `javascript` as they're technically JSON5 files?

Reply:
There is a json5 option too, I think, used it on the results docs

Reply:
This particular one is .json, I've changed the others to json5 - didn't realise that was accepted!